app/helpers/application_helper.rb

```diff
@@ -1,2 +1,26 @@
 module ApplicationHelper
+  class << self
+    def reduce_same_keys(arr)
+      remove_id_from_xml(arr)
+      grouping_data_from_file(arr).each_with_index.map do |a, index|
```

Comment:
Why is `.each_with_index` used?

Reply:
@matija Good catch! Previous this method looks different and the index was used.